### PR TITLE
Info about code coverage report

### PR DESCRIPTION
**Pull request by tisnik**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Description

Info about code coverage report (documentation update)

## Related Tickets & Documents

- Related Issue #[OLS-186](https://issues.redhat.com//browse/OLS-186)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- N/A

### CODE REVIEW

**Comment by bparees** [comment]
/approve

@onmete can lgtm after his question is resolved

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -62,6 +62,9 @@ to enable it automatically).
 
 During testing, code coverage is measured. If the coverage is below defined threshold (see `pyproject.toml` settings for actual value), tests will fail. We measured and checked code coverage in order to be able to develop software with high quality.
 
+Code coverage reports are generated in JSON and also in format compatible with _JUnit_. It is also possible to call the make target `coverage-report` to generate code coverage reports in form of interactive HTML pages. These pages are stored in `htmlcov` subdirectory. Just open index page from this subdirectory in your web browser.
```
[the remark this answers is not in the document]
Comment (tisnik) [reply CONTRIBUTING.md:65]: okay, will do it

[the remark this answers is not in the document]
Comment (tisnik) [reply CONTRIBUTING.md]: done @onmete

**Pull request merged**